src/logger.ts

```diff
@@ -148,7 +148,10 @@ function getPrefixedLogger(prefix?: string): PrefixedLogger {
             // its methodFactory. (An example extension is: storing each log to a rageshake db)
             childLogger.methodFactory = prefixLogger.methodFactory;
             // Rebuild the child logger with the new methodFactory.
-            childLogger.rebuild();
+            // childLogger.rebuild();
+            // TODO: As a stop-gap we use the deprecated method of rebuilding the logger
+            // by setting the level. Complement Crypto uses an older version of loglevel.
```

Comment:
Should we instead update the dependencies in complement crypto?

Reply:
I have done this in: https://github.com/matrix-org/complement-crypto/pull/173

Reply:
Thanks!